src/custom-logger/CustomLogger.ts

```diff
@@ -0,0 +1,11 @@
+import { Logger } from '@nestjs/common';
+
+export class CustomLogger extends Logger {
```

Comment:
kompletnie nie o to mi chodzilo

Reply:
nie ma sensu implementowac tego loggera tak dlugo jak wy tak naprawde chcecie po prostu wrzucic do konsoli cos. Loger sam z siebie daje wam juz timestamp jak sie nie myle i miejsce gdzie zostal uzyty. To jest do wywalenia

Reply:
w sumie to jest tylko do pokazania większej ilości informacji o błędzie. ES2022 dodaje do Errora metode casue, która podaje bardziej szczegółowe informację na temat błędu

Reply:
wg mnie prędzej czy później "cause" będzie trzeba użyć, bo jak jest wielowastwowa struktura i w każdej warstwie jest try-catch a błąd rzuca najbardziej szczegółowa warstwa, to bez cause nie wiadomo gdzie, tylko mamy albo przekazany błąd z samego spodu i koniec albo rzucony błąd z samej góry i koniec. Cause został wprowadzony do ES2022 właśnie po to, aby ułatwić i widać wtedy jak leciał błąd przez wszystkie warstwy. Stack w błędzie JS kompletnie tego nie pokazuje.

Reply:
wywalone